helpers/DeployHelper.ts

```diff
@@ -575,11 +577,14 @@ export class DeployHelper {
       case "v3":
         g16VerifierContractWrapperName = "Groth16VerifierV3Wrapper";
         break;
+      case "authV2":
+        g16VerifierContractWrapperName = "Groth16VerifierAuthV2Wrapper";
```

Comment:
We have all the contract names in the constants. Please reuse them everywhere in this method